crates/bevy_input/src/keyboard.rs

```diff
@@ -650,3 +653,795 @@ pub enum KeyCode {
     /// General-purpose function key.
     F35,
 }
+
+/// Contains the platform-native logical key identifier
+///
+/// Exactly what that means differs from platform to platform, but the values are to some degree
+/// tied to the currently active keyboard layout. The same key on the same keyboard may also report
+/// different values on different platforms, which is one of the reasons this is a per-platform
+/// enum.
+///
+/// This enum is primarily used to store raw keysym when Winit doesn't map a given native logical
```

Comment:
keysym?

Reply:
I believe it's short for key symbol.

Reply:
Those descriptions are coming mostly from winit, I'm not sure referring to them as keysym helps much the user indeed. A google search on that term doesn't respond with a clear description, so I assume it's a vague term for referring to key codes, or logical key code names.

I'm not opposed to remove the term. Or maybe bikeshed that in another issue

Reply:
@alice-i-cecile another question on that line is the reference to winit which we should probably avoid within bevy_input in favor of "window backend" or equivalent.

Reply:
Yeah, "windowing backend" is slightly better.

Reply:
at the same time, I'm curious how many users don't use winit ; blurring the line of the dependency to winit comes at a price of increased complexity: the user has to understand what is the "windowing backend".

It can be argued it's a "small complexity", or an "implementation detail", but I hate when I have to jump through a lot of "soft references", I'll consider this "extreme abstraction" shadowing winit controversial.

It's listed in the todo on https://github.com/bevyengine/bevy/issues/11052 so this discussion will come again :)